projects/packages/my-jetpack/_inc/components/product-card/index.tsx

```diff
@@ -194,68 +177,39 @@ const ProductCard = inprops => {
 				children
 			) }
 
-			<div className={ styles.actions }>
-				<div className={ styles.buttons }>
-					{ secondaryAction && secondaryAction?.positionFirst && (
-						<SecondaryButton { ...secondaryAction } />
-					) }
-					<ActionButton
-						{ ...props }
-						onActivate={ activateHandler }
-						onFixConnection={ fixConnectionHandler }
-						onManage={ manageHandler }
-						onAdd={ addHandler }
-						onInstall={ installStandaloneHandler }
-						onLearnMore={ learnMoreHandler }
-						className={ styles.button }
-						additionalActions={ additionalActions }
-						primaryActionOverride={ primaryActionOverride }
+			{ recommendation && <RecommendationActions slug={ slug } /> }
+			{ ! recommendation && (
```

Comment:
Another nitpick, instead of two { condition && component } we could have a ternary like { recommendation ? <RecommendationActions /> : (all that markup below) }

Reply:
True, it was cleaner that way when I was experimenting with the solution, but it's good to combine it into one condition now.